Cargo.toml

```diff
@@ -17,21 +17,23 @@ ahash = "0.8.3"
 bincode = "1.3.3"
 napi = {version = "2.13.3", default-features = false, features = ["napi8", "serde-json", "experimental"]}
 napi-derive = {version = "2.13.0", default-features = false}
-polars-core = {git = "https://github.com/pola-rs/polars.git", rev = "ec0c91f93fcd1ac355c667d6c3c3f30b257ea0a6", default-features = false}
-polars-io = {git = "https://github.com/pola-rs/polars.git", rev = "ec0c91f93fcd1ac355c667d6c3c3f30b257ea0a6", default-features = false}
-polars-lazy = {git = "https://github.com/pola-rs/polars.git", rev = "ec0c91f93fcd1ac355c667d6c3c3f30b257ea0a6", default-features = false}
-polars-ops = {git = "https://github.com/pola-rs/polars.git", rev = "ec0c91f93fcd1ac355c667d6c3c3f30b257ea0a6", default-features = false}
-thiserror = "1.0.47"
+polars-core = {git = "https://github.com/pola-rs/polars.git", version = "0.34.2", default-features = false}
+polars-io = {git = "https://github.com/pola-rs/polars.git", version = "0.34.2", default-features = false}
+polars-lazy = {git = "https://github.com/pola-rs/polars.git", version = "0.34.2", default-features = false}
+polars-ops = {git = "https://github.com/pola-rs/polars.git", version = "0.34.2", default-features = false}
+thiserror = "1"
 smartstring = { version = "1" }
 serde_json = {version = "1"}
 either = "1.9"
+rayon = "1.8"
+hashbrown = { version = "0.14", features = ["rayon", "ahash"] }
```

Comment:
why did we need to add these?

Reply:
I was getting an error hence added them, now they are not needed. I will remove them.